bundles/org.openhab.binding.plclogo/src/main/java/org/openhab/binding/plclogo/internal/handler/PLCBridgeHandler.java

```diff
@@ -256,20 +295,26 @@ public void dispose() {
         logger.debug("Dispose LOGO! bridge handler.");
         super.dispose();
 
+        final var rtcJob = this.rtcJob;
         if (rtcJob != null) {
             rtcJob.cancel(false);
-            rtcJob = null;
-            logger.info("Destroy RTC job for {}.", config.get().getAddress());
+            logger.info("Destroy RTC job for {}.", config.getAddress());
```

Comment:
or even trace
```suggestion
            logger.debug("Destroy RTC job for {}.", config.getAddress());
```

Reply:
It's on purpose: I run into situations, where one job was not destroyed properly. May be it was a bug in a core...

Reply:
Will not change: This is not a debug. Logo! may go crazy, if more then one connection made. This is only one way to get info, why Logo! got offline: compare time stamps of log entries. Sometimes Logo! network interface crashes too: See readme for details. And no, i do not use UI for anything.

Reply:
Not sure what the UI has to do with this?

You seem to describe a perfect debugging situation. I'm trying to understand why this is so important to you. The user's log will be filled with these log lines and what should a user do with it? If they have trouble, we usually ask them to change the log lever to debug and from there report the log so we as developers can fix the issue.